utils/zerovec/src/varzerovec/databake.rs

```diff
@@ -14,11 +14,46 @@ impl<T: VarULE + ?Sized> Bake for VarZeroVec<'_, T> {
     }
 }
 
-impl<T: VarULE + ?Sized> Bake for &VarZeroSlice<T> {
+impl<T: VarULE + ?Sized> Bake for VarZeroSlice<T> {
     fn bake(&self, env: &CrateEnv) -> TokenStream {
         env.insert("zerovec");
         let bytes = self.as_bytes();
         // Safe because self.as_bytes is a safe input
         quote! { unsafe { ::zerovec::VarZeroSlice::from_bytes_unchecked(&[#(#bytes),*]) } }
     }
 }
+
+impl<T: VarULE + ?Sized> Bake for &VarZeroSlice<T> {
+    fn bake(&self, env: &CrateEnv) -> TokenStream {
+        env.insert("zerovec");
+        let bytes = self.as_bytes();
+        // Safe because self.as_bytes is a safe input
+        quote! { unsafe { &::zerovec::VarZeroSlice::from_bytes_unchecked(&[#(#bytes),*]) } }
+    }
+}
+
+#[test]
```

Comment:
Request: Add `test_bake!` on the derived structs in *yoke_impls.rs*. You can make them all empty to save on boilerplate. I just want to see that the derives work as well as the direct baking.